### ISSUE TITLE
Allow selfsigned SSL certificates

### ISSUE THREAD

**Issue by scharel** [scharel/harbour-nextcloudnotes#32]
This should be an option per account.
The here: https://stackoverflow.com/questions/35577243/access-and-read-html-php-page-by-https-request-by-qml-js-behind-htaccess-apache2

    #include <QSslConfiguration>
    #include <QSslSocket>

    QSslConfiguration config = QSslConfiguration::defaultConfiguration();
    config.setPeerVerifyMode(QSslSocket::VerifyNone);
    QSslConfiguration::setDefaultConfiguration(config);`

**Comment by vbelloir** [comment]
Hi,
Is there any update?
My owncloud is hosted on a Synology NAS, which is accessed with a Let's Encrypt certificat.

App simply tells me that an error occurred.

If I can help.

**Comment by dimpflmoser** [comment]
Hi,
it seems to work for me:
My Nextcloud runs on an Odroid server at home. That webserver (apache) hosting my Nextcloud instance is also equipped with a Let's Encrypt certificate and I use scharel's Nextcloud notes on an encrypted channel without any issues.

Actually since you use a Let's Encrypt certificate it's not self-signed and should be trusted by on of the certificate chains installed on your local SSL installation.

**Comment by vbelloir** [comment]
You are right. My issue doesn't concern self certificat, by custom https port I think

**Comment by dimpflmoser** [comment]
My home server also runs on a custom port, which doesn't cause any cert trust issues with Nextcloud notes.

So I assume it's something different.
Have you checked the URL stored in Nextcloud notes in a ordinary web browser?

**Comment by vbelloir** [comment]
I should create another issue, because my behavior seems to be note related to this one.
